.github/release-drafter.yml

```diff
@@ -29,6 +29,9 @@ categories:
     collapse-after: 3
     labels:
       - "dependencies"
+  - title: "🔨 Internal development"
+    labels:
+      - "dev"
```

Comment:
Thoughts on putting this above the dependencies? I see this as something that at least _some_ people would want to look at, whereas no one cares about the dependencies

Reply:
sure 😅👍